src/components/MaterialUI/Inputs/Select.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import MenuItem from "@material-ui/core/MenuItem";
+import SelectMUI from "@material-ui/core/Select";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => ({
+	select: {
+		"& ul": {
+			//top: '20px',
+			minWidth: "175px",
+			maxHeight: "300px",
+		},
+		"& li": {
+			fontSize: 12,
+			"&:hover": {
+				backgroundColor: theme.palette.primary.main,
+			},
+		},
+	},
+	input: {
+		borderRadius: 4,
+		position: "relative",
+		backgroundColor: theme.palette.background.paper,
+		border: "1px solid #ced4da",
+		fontSize: 12,
+		minWidth: "150px",
+		maxWidth: "200px",
+		padding: "6px 6px 6px 6px",
+		transition: theme.transitions.create(["border-color", "box-shadow"]),
+		"&:focus": {
+			borderRadius: 4,
+			borderColor: "#80bdff",
+			boxShadow: "0 0 0 0.2rem rgba(0,123,255,.25)",
+		},
+	},
+}));
+
+const MenuProps = {
+	getContentAnchorEl: null,
+	anchorOrigin: {
+		vertical: "bottom",
+		horizontal: "left",
+	},
+};
+
+const ChevronDown = props => (
+	<div
+		{...props}
+		style={{
+			fontSize: "32px",
+			fontFamily: "courier,monospace",
+			//fontWeight: 'bold',
+			color: "black",
+			height: "10px",
+			width: "20px",
+			marginRight: "5px",
+			padding: "12px 0 0",
+		}}
+	>
+		ˇ
+	</div>
+);
+
+const Select = ({ options, update, value }) => {
```

Comment:
I think it's better to create selectProps and move options and value there. Sample of that you can find in expansionPanelProps for Expansion Panel. Then we have to pass in wrapper just selectProps object which will include options/value props (and more props probably in future). By this approach we will avoid necessity to rewrite Select component signature (like properties) in future what will allow us to avoid necessity to adjust all the usages of it in OMS.

Reply:
I change the props to have only one, however, I don't see the point to have it strong typed as 99% of the code is not anyway in orc-shared.

Reply:
The way how I've implemented componentProps allows final user to use only material ui docs and intellisense without going to orc-shared and checking which keys he have to use for properties and how they will be used in wrapper. So, the point is usability and unification.